docs/source/user_guide/configuration_options.rst

```diff
@@ -0,0 +1,890 @@
+==============================
+Receptor Configuration Options
+==============================
+
+---------------------
+Receptor command line
+---------------------
+
+Command line arguments have the form: receptor [--<action> [<param>=<value> ...] ...]
+
+<action> are listed below.  When an action has parameters, they are listed in their own section.
+
+-------------
+Miscellaneous
+-------------
+
+.. list-table:: Miscellaneous
+    :header-rows: 1
+    :widths: auto
+
+    * - Action
+      - Description
+    * - --bash-completion
+      - Generate a completion script for the bash shell
+    * - --config <filename>
+      - Load additional config options from a YAML file
+    * - --control-service
+      - Run a control service
+    * - --help
+      - Show this help
+    * - --local-only
+      - Run a self-contained node with no backends
```

Comment:
```suggestion
      - Runs a self-contained node with no backend.
```

Reply:
"With no backend" - does this mean with no persistence? If so, I would prefer to spell out what that means. Maybe just describe it as ephemeral.

Reply:
It means it runs without any backend (below the mesh - tcp/udp/websocket) processes.  So it can only talk to itself.

Reply:
Sounds good to me. Thanks for explaining!